### ISSUE TITLE
[Feature Request] Show Moved Lines of Code

### ISSUE THREAD

**Issue by programming-wolf** [ForkIssues/Tracker#1443]
Since 2018, Git supports a new output for the diff, `--color-moved=zebra` and `--color-moved=dimmed-zebra`, respectively. This will highlight lines of code that have only been moved differently, with a dark and subtle colour. 
It would be amazing to have this feature in Fork, to see if/when large chunks of code have simply been moved, but not changed.

**Comment by ravicious** [comment]
Yes please! To show you how good it'd be, here's `git diff` with the following config:

```
  colormoved = "default"
  colormovedws = "allow-indentation-change"
```

In the screenshot below, it's immediately clear that I moved the whole block and added only `this.initTshdClient`.

<img title="Colormoved" width="550" src="https://github.com/fork-dev/Tracker/assets/27113/7aa693b1-be9b-486b-bbaa-c6e8e79e7c6a"/>

**Comment by ravicious** [comment]
Edit: I just noticed that the OP mentions `dimmed-zebra`. Leaving this comment anyway to show an example of how it looks like.

I think I've reached to maintainers over email a while ago about this and the response was that this would introduce too many colors and would not be readable.

Another option would be to use `dimmed-zebra` which introduces only a single extra color.

https://ruempler.eu/2023/12/31/code-reviews-with-git-color-diffs/

<img title="dimmed zebra example" width="550" src="https://ruempler.eu/2023/12/31/code-reviews-with-git-color-diffs/example.png"/>
